Flow.Launcher/MainWindow.xaml

```diff
@@ -83,24 +84,29 @@
                      AllowDrop="True"
                      Visibility="Visible"
                     Background="Transparent"
-                    Margin="18,0,56,0">
+                    Margin="16,0,56,0">
                         <TextBox.ContextMenu>
                             <ContextMenu>
                                 <MenuItem Command="ApplicationCommands.Cut"/>
                                 <MenuItem Command="ApplicationCommands.Copy"/>
                                 <MenuItem Command="ApplicationCommands.Paste"/>
                                 <Separator />
-                                <MenuItem Header="Settings" Click="OnContextMenusForSettingsClick" />
+                                <MenuItem Header="{DynamicResource flowlauncher_settings}"  Click="OnContextMenusForSettingsClick" />
                             </ContextMenu>
                         </TextBox.ContextMenu>
                     </TextBox>
-                    <svgc:SvgControl Source="{Binding Image}" HorizontalAlignment="Right" Width="42" Height="42"
-                                     Background="Transparent"/>
+                    <svgc:SvgControl Source="{Binding Image}" HorizontalAlignment="Right" Width="25" Height="25" Margin="0,2,18,0" Opacity="0.2" Background="Transparent" Style="{DynamicResource SearchIconStyle}"/>
```

Comment:
same screenshot

Reply:
![image](https://user-images.githubusercontent.com/6903107/134943425-409b9426-72e9-4249-ac02-c6895b57cbc7.png)
It may not be seen well in some subjects, but it has been the same before. In the future, I will try to make it visible according to the theme.

Reply:
Probably make this editable and edit it in theme file instead of the MainWindow.xaml